web/src/games/picnicGo/game.ts

```diff
@@ -200,7 +200,9 @@ export function confirmScore(g: IG, ctx: Ctx) {
 
 export const PicnicGoGame = {
   name: 'picnicGo',
-
+  // the statistical distribution of cards on Freeboardgames.org seems to be strange,
+  // so test a different seeding approach (instead of the BGIO default based on Date.now):
+  seed: Math.random(),
```

Comment:
This will seed _all_ matches with a single seed, which is probably not what you want. In an FBG context this is actually tricky to do probably as you'd need a game factory to generate the game object each time a match is created.

Reply:
Ah, you are correct. I fixed it in a467d9a